example.py

```diff
@@ -507,6 +507,7 @@ def _next_timestamp() -> datetime:
 extracted_file_file_facet1 = uco.observable.FacetFile(
     file_extension="jpg",
     file_name="IMG_0123.jpg",
+    file_path="file/Image/IMG_0123.jpg",
```

Comment:
Here's the `example.py` line that should be cut.

Separately, I noticed something:  There's another reference in `example.py` to a `FileFacet` with `filePath` `/sdcard/ImG_0123.jpg`, but `fileName` `IMG_0123.jpg`---note the capitalization difference.  Was that intentional?

Reply:
`IMG_0123.jpg` seems to have been spelled a few places in the graph as `ImG_0123.jpg`.  It looks like a carried-around copy-paste error.  Please correct this if it was an error.

Reply:
@ajnelson-nist I changed the example.py. As far as the Tool class is concerned, please push the changes you deem appropriate. Thanks for your patience!